source/simulator/stokes_matrix_free.cc

```diff
@@ -1325,30 +1440,51 @@ namespace aspect
 
     for (unsigned int level=0; level<n_levels; ++level)
       {
-        // Create level viscosity table
-        {
-          const unsigned int n_cells = mg_matrices_A_block[level].get_matrix_free()->n_macro_cells();
-          level_viscosity_tables[level].reinit(TableIndices<1>(n_cells));
+        // Create multilevel viscosity tables. For DGQ0, this is one value per cell,
+        // for DGQ1 this is n_q_points values per cell.      {
```

Comment:
what is this opening ``{``?

Reply:
This is left over from before I removed excess brackets. Just a typo.